src/services/FileSystemService.ts

```diff
@@ -54,47 +67,35 @@ export class FileSystemService implements IProjectFilesBrowserService {
     return stats.isDirectory();
   }
 
-  async isSymbolicLink(path: string) {
-    const stats = await fs.promises.lstat(path);
-    return stats.isSymbolicLink();
-  }
-
   async getMetadata(path: string): Promise<Metadata> {
     if (!(await this.exists(path))) {
       throw ErrorFactory.newInternalError(`File doesn't exist (${path})`);
     }
 
-    const extension = nodePath.posix.extname(path);
```

Comment:
> To achieve consistent results when working with POSIX file paths on any operating system, use path.posix 
[https://nodejs.org/api/path.html#path_windows_vs_posix](url)

I'm not sure about the part of the sentence ...with POSIX file.. but.. How does it work without the .posix? Check the example in the url.

Reply:
POSIX is for UNIX system, not for Windows